### DISCUSSION TITLE
How does my university provide the Developer's Pack to Students?

### DISCUSSION THREAD

**Discussion by djohnson62** [community/community#24134] [GitHub Education]
<p>Hello all,</p>
<p>First post, please be gentle.</p>
<p>I understand that students can apply for the Student Developer Pack individually, which is great.</p>
<p>However, if my university applies for the GitHub Campus Program, will we be able to give students access to the Student Developer Pack? I see one of the benefits of the Campus Program is that university gets “automatic” access to the Developer Pack.</p>
<p>What does “automatic” mean?</p>
<p>Thank you so much in advance for your answer.</p>
<p>David P.</p>

**Comment by Zerotask** [comment]
<p>I just entered my student address and the name of my university to get the Student’s Developer Pack.</p>
<p>The process of applying is described here:</p><aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/apply-for-a-student-developer-pack" target="_blank" rel="noopener nofollow ugc">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/apply-for-a-student-developer-pack" target="_blank" rel="noopener nofollow ugc">Apply for a student developer pack - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/apply-for-a-student-developer-pack" target="_blank" rel="noopener nofollow ugc">//docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/apply-for-a-student-developer-pack</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by djohnson62** [comment]
<p>Great. Thank you.</p>
<p>Do you know if your university is a part of the GitHub Education Campus Program? This may be what “automatic” means.</p>

**Comment by Zerotask** [comment, marked as answer]
<p>As far as I know my University isn’t partnered with GitHub and If I Understand the docs correctly, you can’t apply manually If your University is a partner. Then your email is directly part of that program</p>

**Comment by djohnson62** [comment]
<p>Excellent. I think that is the answer I was looking for.</p>
<p>Thank you!</p>
